elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/AggregationAndSortTest.java

```diff
@@ -412,16 +412,62 @@ private static Connection createConnection() throws SQLException {
    */
   @Test void testGroupTextField() {
     CalciteAssert.that()
-        .with(AggregationTest::createConnection)
+        .with(AggregationAndSortTest::createConnection)
         .query("select cat6, count(1) as CNT from view group by cat6")
         .returnsUnordered("cat6=null; CNT=2",
             "cat6=text1; CNT=1");
 
     CalciteAssert.that()
-        .with(AggregationTest::createConnection)
+        .with(AggregationAndSortTest::createConnection)
         .query("select cat1, cat6 from view group by cat1, cat6")
         .returnsUnordered("cat1=a; cat6=null",
             "cat1=b; cat6=null",
             "cat1=null; cat6=text1");
   }
+
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4860">[CALCITE-4860]
+   * In Elasticsearch adapter, support NULLS FIRST and NULLS LAST query</a>.
+   */
+  @Test void testNullsSort() {
+    CalciteAssert.that()
```

Comment:
I don't put this test to ElasticSearchAdapterTest is because we need some null values, which may cause many unnecessary changes since we need to modify the zips-mini.json.